docs/users-guide/08-dashboard-filters.md

```diff
@@ -4,84 +4,97 @@
+Each card will feature a dropdown menu where you can select the column to filter. The sidebar on the right displays the settings for the current filter. If there’s a card on your dashboard that you don’t want to use with the filter, or that it doesn’t make sense to use with the filter, that’s okay — the filter will only be applied to the cards you select.
+
+So here’s what we’re doing — when we pick a month and year with our new filter, the filter needs to know which column in the card to filter on. For example, if we have a **Total Orders** card, and each order has a `Date Ordered` as well as a `Date Delivered` column, we have to pick which of those columns to filter — do we want to see all the orders _placed_ in January, or do we want to see all the orders _delivered_ in January? So, for each card on our dashboard, we’ll pick a date column to connect to the filter. If one of your cards says there aren’t any valid columns, that just means that card doesn’t contain any columns that match the kind of filter you chose.
 
 ![Select fields](images/dashboard-filters/04-select-fields.png)
 
-Before we click the `Done` button at the top of the screen, we can also customize the label of our new filter by clicking on the pencil icon next to it. We’ll type in a new label and hit enter. Now we’ll click `Done`, and then save the changes to our dashboard with the `Save` button.
+Before we **Save** our changes, we can use the right sidebar to customize the **Label** of our new filter, or set a **Default value**.
 
-![Edit the filter label](images/dashboard-filters/05-edit-label.png)
+When you're finished wiring up the filter, click **Done** at the bottom of the sidebar, then click on **Save** in the top right to save the dashboard.
 
 ### Editing a filter
 
-To edit a filter, enter dashboard editing mode, then click the `Edit` button on the filter you want to change. You an also click `Remove` to get rid of a filter. If you do this by accident, just click `Cancel` in the top-right to exit dashboard editing mode without saving your changes. To reorder your filters, just click on the grabber handle on the left side of a filter and drag it to a different position.
-
-![Edit or remove a filter](images/dashboard-filters/06-edit-and-remove.png)
-
-### Setting a default value
-
-If you want one of your filters to start with a default value when you load the dashboard it’s in, while in filter editing mode just click on the filter to select a value. Click the blue X if you want to remove the default value.
-
-![Set a default value](images/dashboard-filters/07-default-value.png)
+- **To edit a filter**: click the **pencil** icon to enter dashboard editing mode, then click the **gears** icon button on the filter you want to change. 
```

Comment:
Maybe we should mention how this is also how you get back to the mode where you can update/edit the mappings between a filter and the cards in the dashboard.